app/graphql/types/record_type.rb

```diff
@@ -108,7 +108,7 @@ class AlternateTitleType < Types::BaseObject
 
   # Warning: related_place was supposed to be an array but was incorrectly a string in graphql for v1
   class LocationType < Types::BaseObject
-    field :geopoint, String, description: 'GeoPoint data for the location, if applicable'
```

Comment:
As we discussed on zoom, let's try to deprecate `geopoint` as a matter of best practice even though it is very unlikely anyone is using it.

So the we'd mark the field as deprecated and suggest using the new field while still mapping the new field value to the old field so people still get what they asked for either way. See https://github.com/MITLibraries/timdex/blob/main/app/graphql/types/record_type.rb#L34-L36 for an example of how we have done that on other fields that got renamed.

Reply:
Sure thing - I've switched from a replacement to a deprecation for `geopoint` now. The test has been updated to include it as well. I see the deprecation announcement in the help docs, but don't see it in the actual API response, so I don't yet have a way to test for the deprecation itself.

Reply:
I don't think we need an explicit test for the deprecation at this point. Manually confirm it is flagged as deprecated along with the tests that both fields return the expected data should be solid. I'll finish up the review on this now thanks!